documentation/docs/components/columns/gap.md

```diff
@@ -4,5 +4,106 @@ title: Gap
 sidebar_label: Gap
 ---
 
+import Columns from 'react-bulma-components/lib/components/columns'
+import Hero from 'react-bulma-components/lib/components/hero'
+import Section from 'react-bulma-components/lib/components/section';
+
+---
+## **Import**
+
+```js
+import Columns from 'react-bulma-components/lib/components/columns'
+```
+
+---
+---
+
+
+## **Examples**
+<Section>
+<Columns
+      variableGap={{
+        mobile: 1,
+        tablet: 0,
+        desktop: 3,
+        widescreen: 8,
+        fullhd: 2,
+      }}
+    >
+  <Columns.Column size={3}>
+    <Hero color="dark">Size 1</Hero>
+  </Columns.Column>
+  <Columns.Column size={3}>
+    <Hero color="light">Size 1</Hero>
+  </Columns.Column>
+  <Columns.Column size={3}>
+    <Hero color="white">Size 1</Hero>
+  </Columns.Column>
+  <Columns.Column size={3}>
+    <Hero color="danger">Size 1</Hero>
+  </Columns.Column>
+</Columns>
```

Comment:
Please indent with 2 spaces. Also, the `varableGap` prop was indented with 2 extra spaces. Please remove them.